src/index.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable quotes */
+/* eslint-disable no-unused-vars */
+import _ from "lodash";
+import "./style.css";
+
+const body = document.getElementById("todos-list");
+let taskno = 0;
+let taskLists = [];
+
+function printfn() {
```

Comment:
- [ ] Nice to have a function to iterate over the tasks array and populate an HTML list item element for each task. :+1: but please from the project requirements, you should set an array of some simple *to do tasks* (array of objects). Each task object should contain three keys: 
  1. description [`string`].
  2. completed [`bool`].
  3. index: [`number`]. 
  
**But for now, you have empty array of tasklists.* :smile: